### ISSUE TITLE
Terraform github_organization doesn't allow for retrieval of actual org identifier, only Descriptive one, if present.

### ISSUE THREAD

**Issue by mykelalvis** [integrations/terraform-provider-github#709]
### Terraform Version
```
Terraform v0.14.7
+ provider registry.terraform.io/hashicorp/github v4.5.0
+ provider registry.terraform.io/hashicorp/local v2.1.0
```
### Affected Resource(s)
- `github_organization`
If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.
### Terraform Configuration Files
```hcl
data "github_organization" "somewhereorg" {
    name = "somewhere"
}
```
### Expected Behavior
I'd expect to be able to retrieve the string `somewhere` from `data.github_organization.somewhereorg.orgname` (or some similar attribute (`orgslug`?  I dunno)
### Actual Behavior
The only viable outputs are `.id` (an integer) and `.name` which is the _descriptive text_ of the organization (e.g "The Somewhere Organization")
### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Make a `github_organization` data resource as above.
2. Attempt to use that as the `template.owner` of a `github_repository` resource
3. Be sad.

**Comment by jcudit** [comment]
> Be sad.

💔

**Comment by Kavinraja-G** [comment]
Is this still neeede ? @jcudit  I would like to work on this issue. Thanks! :)

**Comment by jcudit** [comment]
@Kavinraja-G yes, this does not conflict with other feature development.  Feel free to link a PR to this issue when ready for review and THANK YOU!

(sorry for the late reply)

**Comment by pallxk** [comment]
`data.github_organization.somewhereorg.login`, the login of the organization account, should be what you need.

**Comment by kfcampbell** [comment]
This should now be fixed in #1052 and release [v4.23.0](https://github.com/integrations/terraform-provider-github/releases/tag/v4.23.0).
